libraries/Microsoft.Bot.Builder.Dialogs/Extensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using System;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs
+{
+    /// <summary>
+    /// Extension method on object <see cref="object"/>.
+    /// </summary>
+    internal static partial class Extensions
+    {
+        /// <summary>
+        /// Extension Method on object to cast to type T to support TypeNameHandling.None during storage serialization.
+        /// </summary>
+        /// <param name="obj">object to cast.</param>
+        /// <typeparam name="T">type to which object should be casted.</typeparam>
+        /// <returns>T.</returns>
+        public static T CastTo<T>(this object obj)
```

Comment:
Should add unit tests to cover this extension

Reply:
I didn't add this...it's just in 4.7 branch :p

---
In reply to: [375486985](https://github.com/microsoft/botbuilder-dotnet/pull/3320#discussion_r375486985) [](ancestors = 375486985)